docs/validation.md

```diff
@@ -46,14 +46,24 @@ This will
 - Run the tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to Unix: `unix:///var/run/dockershim.sock` or Windows: `tcp://localhost:3735` by default. For other runtimes, the endpoint can be set by flags `--runtime-endpoint` and `--image-endpoint`.
+For specific runtimes, the endpoint can be set by flags
+`--runtime-endpoint` and `--image-endpoint`. critest connects to Unix:
```

Comment:
Again, this "default endpoint" is probably not true. Also critest should search all of them.

Reply:
Not sure what the correct one is here then. Revert to dockershim.sock?

Reply:
Can't you just leave it out ?

For specific runtimes, the endpoint can be set by flags `--runtime-endpoint` and `--image-endpoint`.